apps/protailwind/studio/sanity.json

```diff
@@ -14,7 +14,8 @@
     "@sanity/desk-tool",
     "taxonomy-manager",
     "@sanity/code-input",
-    "cloudinary"
+    "cloudinary",
+    "@sanity/vision"
   ],
   "env": {
     "development": {
```

Comment:
I think the plugin can be removed from development env now. Also please don't forget to run `yarn` and commit yarn lock file.